controllers/dashboard_test.go

```diff
@@ -163,6 +163,35 @@ var _ = Describe("DashboardController", func() {
 					Expect(len(matches)).To(BeZero())
 				})
 			})
+
+			Context("given `keyword` param is set", func() {
+				It("shows only `keyword` filter records", func() {
+					keywordFilter := "keyword_keyword_keyword"
+					user := FabricateUser(faker.Email(), faker.Password())
+					_ = FabricateKeyword(keywordFilter, faker.URL(), 0, user)
+					_ = FabricateKeyword(fmt.Sprintf("%v %v %v", faker.Word(), keywordFilter, faker.Word()), faker.URL(), 0, user)
```

Comment:
At first, it seems like there is no need to build a complex string for the keyword string. It seems over-complexifying 😆 But on close inspection, there are two types of keywords created: a simple keyword string and a complex keyword one. That's where comments can help by naming unused fabricated fixtures:

```suggestion
					// Exact match keyword
					_ = FabricateKeyword(keywordFilter, faker.URL(), 0, user)
					// Fuzzy match keyword
					_ = FabricateKeyword("%v %v %v", faker.Word(), keywordFilter, faker.Word()), faker.URL(), 0, user)
					// non-match keyword
					_ = FabricateKeyword("paragraph_paragraph_paragraph", faker.URL(), 0, user)
```

Think of future developers and also your future self by making code easier to understand.